### PR TITLE
FPGA: New sample `platform_designer_standard`

### PR DESCRIPTION
**Pull request by haoyanwa**
# Adding a New Sample: `platform_designer_standard`

## Description

An Intel® FPGA tutorial demonstrating how to export a reusable IP component to Intel® Quartus® Prime Standard and Platform Designer targeting Cyclone® V SoC FPGAs.

Features include:
- Updated script for building the Quartus system for Quartus® Prime Standard.
- Updated script for board testing, including System Console initialization and establishing a connection to the host.
- Revised tutorial steps, tailored for Quartus Prime Standard Edition, with updated screenshots.

This code sample is tested on a DE1-SoC board using Quartus Prime Standard 23.1std.

## Checklist
Administrative
- [x] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): @whitepau 
- [x] If you have any new dependencies/binaries, inform the oneAPI Code Samples Project Manager

Code Development
- [x] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [x] Adhere to readme template 
- [x] Enforce format via clang-format config file
- [x] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [x] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [x] FPGA Regtest (Linux) https://spetc.intel.com/testsummary?testRunIds=10661900
- [x] FPGA Regtest (Windows) https://spetc.intel.com/testsummary?testRunIds=10661973

Security and Legal
- [ ] OSPDT Approval (see Project Manager for assistance)
- [x] Compile using the following compiler flags and fix any warnings, the falgs are: "/Wall -Wformat-security -Werror=format-security"
- [ ] Bandit Scans (Python only)
- [ ] Virus scan

Review
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth(@tomlenth) and/or Project Manager
- [ ] Tested using Dev Cloud when applicable

### CODE REVIEW

**Comment by tomlenth** [comment]
Very thorough, much like all the PSG sample readmes.  thank you.

File: DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+> - Questa*-Intel® FPGA Starter Edition
+> - Questa* Advanced Simulator
+> - ModelSim® SE
+>
+> To use the hardware compile flow, Intel® Quartus® Prime Standard Edition must be installed and accessible through your PATH.
+
+## Prerequisites
+
+This sample is part of the FPGA code samples.
+It is categorized as a Tier 1 sample that helps you getting started.
```
Comment (yuguen-intel) [inline DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md:26]: If this is a Tier 1, then it is not categorized correctly in the main README, if this is a Tier 3 sample, this section is incorrect
I can see that there is the same issue in the platform_designer sample, so both should be addressed - please synchronize with @whitepau to decide where these samples should live

File: DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/sample.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          "cd ../../..",
+          "mkdir build",
+          "cd build",
+          "cmake -G \"NMake Makefiles\" ../Tutorials/Tools/platform_designer_standard/add_oneapi",
+          "nmake report"
+        ]
+      }
+    ]
+  },
+  "expertise": "Getting Started"
```
Comment (yuguen-intel) [inline DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/sample.json:70]: This should align with the Tier of the sample

**Review by yuguen-intel — CHANGES_REQUESTED** [review]

File: DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md (hunk 1 above)
Comment (whitepau) [reply DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md:26]: Both Platform Designer samples should both be Tier 1 samples.

**Review by yuguen-intel — APPROVED** [review]

**Pull request merged**